app/views/posts/index.html.erb

```diff
@@ -12,3 +13,22 @@
 </div>
 
 <%= link_to "New post", new_post_path %>
+<%end%>
+<%##############################%>
```

Comment:
- [ ] [OPTIONAL] Kindly remove all unused codes/comments in order to have a clean codebase.